packages/sources/cfbenchmarks-test/src/endpoint/common/crypto.ts

```diff
@@ -76,21 +76,19 @@ export const cryptoRequestTransform = (req: AdapterRequest<RequestParams>): void
 
 export const routingTransport = new RoutingTransport(
   {
-    // rest: makeRestTransport('primary'),
-    // restSecondary: makeRestTransport('secondary'),
+    rest: makeRestTransport('primary'),
+    restSecondary: makeRestTransport('secondary'),
     websocket: makeWsTransport('primary'),
     websocketSecondary: makeWsTransport('secondary'),
   },
   (_, config) => {
-    /* [TODO]: once framework level transport selectors are added to request use here to route */
-    // if (config?.API_SECONDARY) {
-    //   if (config?.WS_ENABLED) return 'websocketSecondary'
-    //   return 'restSecondary'
-    // }
+    if (config?.API_SECONDARY) {
+      if (config?.WS_ENABLED) return 'websocketSecondary'
+      return 'restSecondary'
+    }
 
-    // if (config?.WS_ENABLED) return 'websocket'
-    // return 'rest'
-    return config?.API_SECONDARY ? 'websocketSecondary' : 'websocket'
+    if (config?.WS_ENABLED) return 'websocket'
```

Comment:
Remove `?` in the latest version of the EA framework

Reply:
Done